app/src/app/[lng]/organization/[id]/account-settings/account/organization-details-tab.tsx

```diff
@@ -0,0 +1,178 @@
+import { Box, HStack, Text, Input, Icon } from "@chakra-ui/react";
+import { Field } from "@/components/ui/field";
+import { Button } from "@/components/ui/button";
+import { useForm } from "react-hook-form";
+import { z } from "zod";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { MdWarning } from "react-icons/md";
+import { useParams } from "next/navigation";
+import { useTranslation } from "@/i18n/client";
+import { OrganizationResponse } from "@/util/types";
+import { useGetUserAccessStatusQuery, useUpdateOrganizationMutation } from "@/services/api";
+import { UseErrorToast, UseSuccessToast } from "@/hooks/Toasts";
+import { Trans } from "react-i18next";
+import { useMemo } from "react";
+const OrganizationDetailsTab = ({
+  organization,
+}: {
+  organization: OrganizationResponse | undefined;
+}) => {
+  const { lng } = useParams();
+  const { t } = useTranslation(lng as string, "admin");
+
+  const schema = z.object({
+    email: z.string().email("invalid-email").min(1, "required"),
+    name: z.string().min(3, "required"),
+  });
+
+  const { showErrorToast } = UseErrorToast({
+    title: t("error-message"),
+  });
+  const { showSuccessToast } = UseSuccessToast({
+    title: t("organization-updated"),
+    duration: 1200,
+  });
+
+  type Schema = z.infer<typeof schema>;
+
+  const {
+    register,
+    handleSubmit,
+    reset,
+    formState: { errors, isDirty },
+  } = useForm<Schema>({
+    mode: "all",
+    defaultValues: {
+      email: organization?.contactEmail,
+      name: organization?.name,
+    },
+    resolver: zodResolver(schema),
+  });
+
+  const [updateOrganization, { isLoading }] = useUpdateOrganizationMutation();
+  const { data: userAccessStatus } = useGetUserAccessStatusQuery(
+    {},
+  );
+  const handleFormSubmit = async (data: Schema) => {
+    const { name, email } = data;
+
+    // TODO prevent users from editing the default organization
+    const response = await updateOrganization({
+      id: organization?.organizationId as string,
+      name,
+      contactEmail: email,
+    });
+
+    if (response.error) {
+      console.error(response.error);
+      showErrorToast();
+      return;
+    }
+
+    showSuccessToast();
+  };
+
+  const { numCities, totalCities } = useMemo(() =>
+    organization?.projects.reduce(
+      (acc, proj) => ({
+        numCities: acc.numCities + (proj?.cities.length ?? 0),
```

Comment:
```suggestion
        numCities: acc.numCities + (proj?.cities?.length ?? 0),
```
Otherwise this crashes if `project` is null